doc/pages/source/documentation/index.md

```diff
@@ -268,6 +270,38 @@ def self.trigger_task(async, action, *args, &block)
 end
 ```
 
+#### Scheduling
+
+Scheduling an action means setting it up to be triggered at set time in future.
+Any action can be scheduled by calling:
+
+```ruby
+world_instance.schedule(AnAction, start_at, start_before, *args)
+```
+
+This snippet of code would schedule `AnAction` with arguments `args` to be executed
+in the time interval between `start_at` and `start_before`. Setting `start_before` to `nil`
+would schedule this action without the upper time limit.
+
+When an action is scheduled, minimal execution plan is created for it and a schedule record
```

Comment:
Looks like I will have to reword it completely. 

The execution plan is in fact marked as failed (stopped+error) on timeout, the only thing that gets removed is the schedule record.